### PR TITLE
Make runtime initialization for `nodejs` work on Windows

### PR DESCRIPTION
**Pull request by TooTallNate**

### CODE REVIEW

File: src/install-node.ts [hunk 1]
```diff
@@ -28,12 +40,27 @@ export async function installNode(
...
-		res.body
-			.pipe(createGunzip())
-			.pipe(extract({ strip: 1, C: dest }))
-			.on('error', reject)
-			.on('end', resolve);
-	});
+		await pipe(
+			res.body,
+			createGunzip(),
+			extract({ strip: 1, C: dest })
```
Comment (styfle) [inline src/install-node.ts]: Why did you remove `.on('error', reject)`?

Comment (styfle) [reply src/install-node.ts]: I guess you removed the Promise to so this will throw on error?

Comment (TooTallNate) [reply src/install-node.ts]: It wasn't using `promisepipe()` before. That module catches stream error events and throws them.
